src/components/form/FormContainer.jsx

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useState } from 'react';
+import { Heading, Paragraph } from '../shared';
+import { Inputs, TextAreaInput } from '../shared/partials/FormInputs';
+import formimg from '../../assets/images/form-tickets.png';
+import { donation, feeCoverage, initialContent, inputContent, lastPartContent } from '../../data/formInformation';
+import { storeFormData } from '../../firebase/registration';
+import { loggedUser, logUserRegData } from '../../firebase/login';
+
+const FormContainer = () => {
+	const userData = logUserRegData;
+
+	const [isValid, setValid] = useState({
+		recRollNumber: false,
+		name: false,
+		email: false,
+		country: true,
+		state: true,
+		city: true,
+		prefix: true,
+		mobile: true,
+		regType: true,
+	});
+	const [isEmpty, setEmpty] = useState({
+		recRollNumber: true,
+		name: true,
+		email: true,
+		country: false,
+		state: false,
+		city: false,
+		prefix: false,
+		mobile: false,
+		regType: false,
+	});
```

Comment:
Could be made part of the form data state. And the `checkValidity` can be a const representing an expression over the state.

Reply:
the formData was intended to contain mostly placeholders and are constant values. 
but isEmpty, validityCheck are states, so won't it be better if we put them in form itself.